contents/forms/checkbox/checkboxes/metadata.json

```
@@ -0,0 +1,13 @@
+{
+  "en": {
+    "title": "チェックボクシズ",
+    "description": "複数のチェックボックスをグループ化した基本的なチェックボックスグループです。"
+  },
+  "ja": {
+    "title": "Checkboxes",
+    "description": "A basic checkbox group, which groups several checkboxes."
+  },
```

You have the title and description in the wrong order. `Grouped checkboxes` might be better for the title.
```suggestion
{
  "en": {
    "title": "Grouped checkboxes",
    "description": "A basic checkbox group, which groups several checkboxes."
  },
  "ja": {
    "title": "グループ化したチェックボックス",
    "description": "複数のチェックボックスをグループ化した基本的なチェックボックスグループです。"
  },
```

---

OMG, sorry, en and ja were reversed😇

---

I fixed.